registration-processor/pre-processor/registration-processor-introducer-validator-stage/src/main/java/io/mosip/registration/processor/stages/introducervalidator/IntroducerValidator.java

```diff
@@ -253,7 +253,8 @@ private void validateUserBiometric(String registrationId, String userId, List<BI
 
 		} else {
 			List<io.mosip.registration.processor.core.auth.dto.ErrorDTO> errors = authResponseDTO.getErrors();
-			if (errors.stream().anyMatch(error -> error.getErrorCode().equalsIgnoreCase("IDA-MLC-007"))) {
+			if (errors.stream().anyMatch(error -> (error.getErrorCode().equalsIgnoreCase("IDA-MLC-007")
+					|| error.getErrorCode().equalsIgnoreCase("IDA-MLC-018")))) {
```

Comment:
Is this change for MOSIP-13442 ?

I see 2 problems:
1. In case of this scenario, we will get the RID of the parent and it might fail in idRepoService.getUinByRid( call, since it might not find the UIN for the parent RID yet and we might want to handle this
2. If case of child packet is comming with parent UIN, if UIN has a typo instead of rejecting the packet will retry again and again.

Point is applicable for biometric authentication change, supervisor and operator validator change also

Reply:
No Vishwa. 
1. Problem is not with idrepo its with idauth. The fix is provided because when uin generated in idrepo, sometimes same is missing from idauth (because of some websub issue i guess). So this scenario will never occur.
2. In case uin is wrong then idauth will not throw above error. (checked with Loga the error will be thrown if uin is not found). If wrong uin then anyways validation will fail.